vm-setup/roles/firewall/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-use_firewalld: true
+use_firewalld: "{{ lookup('env', 'USE_FIREWALLD') | default(true, true) }}"
```

Comment:
Q: This is set in lib/common.sh based on the OS, and is not user-configurable.  This is thus fixing a bug in firewall setup, where firewalld was used though it shouldn't have been?

Reply:
I have now checked and it was already limited via the same logic as this "use_firewalld" was toggled in 02_configure_host.sh L294.

From practical perspective the only difference is that we are toggling the feature based on the firewalld specific variable not the more generic "DISTRO". I like this method more because it makes it easier to make it compatible with other distros if needed and provides a dedicate toggle switch for the firewall config.

I don't feel like it worth it to remove this firewalld specific env var .